src/setStateIndicator/setIsSick.test.js

```diff
@@ -0,0 +1,40 @@
+import { setIsSick } from "./setIsSick";
+import { time_between_run_function_getRandomNumber_ms } from "../variables/variables";
+
+afterEach(() => {
+  jest.useRealTimers();
+});
+jest.useFakeTimers();
```

Comment:
```suggestion
beforeEach(() => {
  jest.useFakeTimers();
});
 
afterEach(() => {
  jest.useRealTimers();
});

```

Reply:
and all this beforeEach and afterEach should be moved inside `describe` block, becase thus we guarantee that both these beforeEach and afterEach will be executed for each `it` case